src/main/java/commands/ExcludeCommand.java

```diff
@@ -0,0 +1,110 @@
+package commands;
+
+import access.Access;
+import exception.ExclusionFileException;
+import exception.InvalidInputException;
+import storage.Storage;
+import ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+
+public class ExcludeCommand extends Command {
+    String type;
+    public static final String COMMAND_WORD = "exclude";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Excludes or Includes Modules and Chapters from your "
+            + "schedule.\n"
+            + "Parameters: more(excludes from schedule)/less(includes into schedule)\n"
+            + "Example: " + COMMAND_WORD + " more\n"
+            + "Example: " + COMMAND_WORD + " less\n";
+
+
+    public ExcludeCommand(String type) {
+        this.type = type;
+    }
+
+    @Override
+    public void execute(Ui ui, Access access, Storage storage) throws InvalidInputException, ExclusionFileException {
+        switch (type) {
+        case "more":
+            addingToExclusion(ui, storage);
+            break;
+        case "less":
+            removingFromExclusion(ui, storage);
+            break;
+        default:
+            throw new InvalidInputException("The specified type for the exclude command is invalid.\nThe only valid "
+                    + "types are \"more\" and \"less\".");
+        }
+    }
+
+    private void addChapterToExclusion(Storage storage, Ui ui, String type) throws ExclusionFileException,
+            InvalidInputException {
+        String moduleName = ui.getExcludedModuleName(type);
+        String chapterName = ui.getExcludedChapterName(moduleName);

```

Comment:
is it possible to return the thing you want to print to the execute method and call ui from there? For the methods below as well.